II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,11 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
-// be sure to exclude the strings
\ No newline at end of file
+// be sure to exclude the stringsmodule.exports = function (array) {
+    function getAverage(array) {
+        var sum = 0;
+         numbersOnly = array.filter(item=>typeof item == "number").map(item=>item);
```

Comment:
`.map(item=>item)` is redundant, it gives you the same output as the input (except that you create a new array in the process - it may be useful as a "shallow copy" operation but that's not your intent here).